Changes to the ASP.NET Core Razor SDK (#22013)
dotnet/AspNetCore.Docs: aspnetcore/migration/50-to-60.md

+ docker pull mcr.microsoft.com/dotnet/aspnet:6.0
```

## Changes to the ASP.NET Core Razor SDK

The Razor compiler now leverages the new [source generators feature](https://devblogs.microsoft.com/dotnet/introducing-c-source-generators/) to generated compiled C# files from the Razor views and pages in a document. In previous versions:

> * The compilation relied on an individual `RazorGenerate` and `RazorCompile` target to produce the generated code. These targets are no longer valid. In .NET 6, both code generation and compilation are supported by a single call to the compiler.

Author: ```suggestion
* The compilation relied on an individual `RazorGenerate` and `RazorCompile` target to produce the generated code. These targets are not valid in .NET 6. In .NET 6, both code generation and compilation are supported by a single call to the compiler.
```
